### PR TITLE
[WIP] [EOS-25320] Kubernetes main 

### PR DESCRIPTION
**Pull request by VinothVijayakumar-dev**
# Problem Statement
- Merging kubernetes changes on top latest main 

# Design
-  Need to integrate gethostname change on merged libfab code

# Coding
   Checklist for Author
-  This is work in progress

# Testing 
  Checklist for Author
- [ yes] Unit and System Tests are added
- [ yes] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ yes] Testing was performed with RPM

### CODE REVIEW

File: net/libfab/libfab.c.bak [hunk 1]
```diff
@@ -0,0 +1,3641 @@
+/* -*- C -*- */
```
Comment (andriytk) [inline net/libfab/libfab.c.bak:1]: Is this file really needed here?

**Comment by VinothVijayakumar-dev** [comment]
These changes are taken as patch and created a kubernetes branch. Hence closing this PR.
